youtube_dl/extractor/stopgame.py

```diff
@@ -0,0 +1,103 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+import urllib2
```

Comment:
youtube-dl should be compatible with both Python 2 and Python 3. Use symbols from ```..compat``` instead.